src/main/java/org/mule/extension/db/internal/domain/connection/sqlserver/SqlServerConnectionParameters.java

```diff
@@ -138,6 +139,11 @@ public String getUser() {
     return user;
   }
 
+  @Override
+  public java.util.Optional<TlsContextFactory> getTlsContextFactory() {
```

Comment:
Same as Derby

Reply:
No idea, but if it does by implementing this method we would offer the feature.